app/models/package.scala

```diff
@@ -145,6 +145,7 @@ package object models {
                     }
                   }
             }
+        case _ => JsError("Unknown genericc error")
```

Comment:
We know this relates to either an invalid path or value so something like this might be better:
`case (_, _) => JsError("Invalid path or value")`